spec/openapi.yml

```diff
@@ -1124,6 +1177,12 @@ components:
         endedAt: null
         durationMs: null
         args: {'email': 'me@example.com', 'emailOnFailure': 'false', 'emailOnRetry': 'true', 'retries': '1'}
+        facets: {
+          'context': {
```

Comment:
Currently, a run doesn't contain the `context` field (though that was on the roadmap to add). We can omit the `context` in `facets`. (Similar comment for other `runAsRunning `,  `runAsCompleted`, etc)

Reply:
But it does make sense to add it, right?
We do want to be able to add this information here. Are you suggesting an alternative?

Reply:
Im not sure. Is `context` a facet?

Reply:
Based on how the `context` is defined for a job, this is considered a custom facet. Make sense to add it to a run as well 👍